src/api/solr-query.js

```diff
@@ -59,11 +59,23 @@ const buildQuery = (fields, mainQueryField) => fields
   .map((queryFilter) => `fq=${queryFilter}`)
   .join("&");
 
-// Concates all request fields for solr fl parameters 
+// Concates all request fields for solr fl parameters
 const requestField = (fields) => fields
   .map((field) => `${encodeURIComponent(field.field)}`)
   .join(" ")
 
+const dateRangeFacetFieldValue = (dateRangeFacetField) => {
+  var filters = dateRangeFacetField.value;
+  if (filters.length < 2) {
+    filters = [0, 3000];
+  }
+  const dateSuffix = "-01-01T00:00:00Z";
+  // startRange will be year-only, format in the way that makes solr happy
+  let startRangeValue = filters[0] + dateSuffix;
+  let endRangeValue = filters[1] + dateSuffix;
+  return `facet.range=${encodeURIComponent(dateRangeFacetField.field)}&facet.range.gap=${encodeURIComponent("+1YEARS")}&facet.range.start=${startRangeValue}&facet.range.end=${endRangeValue}`;
```

Comment:
Interestingly, even though the field query syntax works with just years, we still have to provide the full date format to solr in order for the facet query to work.  In addition to the facet filters, we need to actually group the dates by year so the UI will show the begin and end year based on the returned data.  This is using the query that @datadavev figured out to get solr to group things by year.